presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -400,6 +401,11 @@ public static PlanMatchPattern limit(long limit, PlanMatchPattern source)
         return node(LimitNode.class, source).with(new LimitMatcher(limit));
     }
 
+    public static PlanMatchPattern tableWriter(PlanMatchPattern source, List<String> columns, List<String> columnNames)
```

Comment:
like in other factory methods in this class, `source` should be the last argument